### PR TITLE
Add W1D3 tutorials

### PR DESCRIPTION
**Pull request by mwaskom**

### CODE REVIEW

File: tutorials/W1D3-ModelFitting/W1D3_Tutorial4.ipynb [hunk 1]
```diff
@@ -0,0 +1,505 @@
+{
```
Comment (bgalbraith) [inline tutorials/W1D3-ModelFitting/W1D3_Tutorial4.ipynb:1]: <p>typo: regression, not regrression</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuromatchAcademy/course-content/pull/104/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (msarvestani) [reply tutorials/W1D3-ModelFitting/W1D3_Tutorial4.ipynb:1]: Thanks. There's a few other little things, I'm looking through.

Comment (mwaskom) [reply tutorials/W1D3-ModelFitting/W1D3_Tutorial4.ipynb:1]: For small things like this we should probably note and handle during post pre pod revisions.

**Comment by msarvestani** [comment]
Somehow day chiefs (and me) missed that 5-7 don't have exercises setup in a
function probably, stubbed, etc. That can't wait, working on it. I'll send
new notebooks here.

**Comment by mwaskom** [comment]
Ok yeah I agree that should block here. 

Can you set the review status to changes required so this PR gets held open?

File: tutorials/W1D3-ModelFitting/W1D3_Tutorial4.ipynb (hunk 1 above)
Comment (bgalbraith) [reply tutorials/W1D3-ModelFitting/W1D3_Tutorial4.ipynb:1]: Generally agree, though this is in the tutorial header in big font, so it's very obvious

**Comment by msarvestani** [comment]
Ugh that's my typo. I added text so all the headers were the same 1-7. We
have to change other issues anyway.

File: tutorials/W1D3-ModelFitting/W1D3_Tutorial4.ipynb (hunk 1 above)
Comment (msarvestani) [reply tutorials/W1D3-ModelFitting/W1D3_Tutorial4.ipynb:1]: New links are up.

**Comment by msarvestani** [comment]
Updated notebooks with bigger figures

**Pull request merged**